templates/report/results.html

```diff
@@ -4,8 +4,8 @@
 
 {% block extra_head %}
     <style>
-        {% for name, color in result_colors %}
-            .cr-{{ name|replace(from=":", to="\:") }} {
+        {% for color in result_colors %}
+            .cr-{{ loop.index0 }} {
```

Comment:
We could save 800kb in the full report by renaming the class to `.r{{ loop.index0 }}`.